docker/debian-bookworm.dockerfile

```
@@ -69,6 +68,17 @@ apt-get clean
 rm -rf /var/lib/apt/lists/*
 _DEPS
 
+#Install Node
+# hadolint ignore=SC1091
+RUN <<_INSTALL_NODE
+#!/bin/bash
+set -e
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash
```

```suggestion
wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash
```

While I would like to keep this on a tag, I don't want to spend time keeping this updated.

---

Done